src/Rules/Extensions/ExtensionMethods.cs

```diff
@@ -8,7 +8,7 @@
 using Axe.Windows.Core.Misc;
 using Axe.Windows.Core.Types;
 using Axe.Windows.Rules.PropertyConditions;
-
+using UIAutomationClient;
```

Comment:
Is this using statement necessary?

Reply:
Yes, because the interop type tagRECT is defined in UIAutomation.